### PR TITLE
Add Gem Wallet to Wallets

### LINKED ISSUE #13206: Suggest a wallet (Gem Wallet)
### Wallet name

Gem Wallet

### What type of wallet?

EOA

### Wallet description

Gem Wallet is an open source self-custodial crypto wallet with support for over 30 blockchains and 14,000+ tokens.

It's user interface comes with a plethora of built-in functionality, such as swaps, staking, name services, Wallet Connect to use DeFi Applications and easy on-ramping to many types of cryptocurrencies.

### Wallet logo

gemwallet

### Background color for brand logo

#2D5BE6

### URL to the project

gemwallet.com

### When did the wallet go live to users?

06/23

### What are the fees users may be paying for using your wallet product?

1) 1% service fee on Fiat to Crypto purchases that go through our partners (ramp, mercuryo and moonpay). These partners will charge there own service fees too, visible at checkout.
2) 0.5% service fee for swapping

### Does the wallet have an active development team?

Yes, we have an active team and open sourced our code: https://github.com/gemwalletcom

### Is the wallet globally accessible?

Yes, our wallet is entirely non-custodial and accessible globally. No KYC requirements.

### Is the wallet available in multiple languages?

English, Spanish, German, French, Chinese, Japanese, Portuguese, Russian, Turkish, Ukrainian, Korean, Arabic, Ukrainian, Vietnamese, Polish

### What social links are there for the project?

Twitter: https://x.com/GemWalletApp
Discord: https://discord.gg/4jpxtwT8r6
Telegram: https://t.me/gemwallet
YouTube: https://www.youtube.com/@gemwallet
Instagram: https://www.instagram.com/GemWalletApp

### Does the wallet have a mobile app? If yes, which operating systems are supported (iOS, Android)?

iOS: https://apps.apple.com/us/app/gem-wallet-crypto-bitcoin/id6448712670
Android: https://play.google.com/store/apps/details?id=com.gemwallet.android

### Does the wallet have a desktop app? If yes, which operating systems are supported (Windows, Mac, Linux)?

Mac: https://docs.gemwallet.com/compatible-devices/#-macos-devices

### Does the wallet have a browser extension? If yes, which browsers are supported (Chromium, Firefox, Safari)?

No

### Is it a hardware wallet?

No

### What is the availability of your source code?

Open source

### What license is the wallet software released under?

GPL-3.0

### Who holds the private keys?

Users are the only ones with access to their public and private keys and can import these with any other wallet they choose

### Please describe the measures taken to ensure the wallet's security and provide documentation wherever possible

Gem Wallet is an open-source Ethereum compatible wallet, and we take security very seriously to ensure the safety and reliability of our users' assets. Here are the measures we have implemented to ensure the wallet's security:

1. **Open Source Transparency:**
    
    Our codebase is fully open-source and available on GitHub [[Gem Wallet GitHub Repository](https://github.com/gemwalletcom)]. This allows the community to review, inspect, and contribute to the security and integrity of the wallet.
    
2. **Support for Community Contributions:**
    
    We maintain a developer Telegram chat to support the community of developers and security enthusiasts that want to come in and help identify and address potential vulnerabilities. Someone from our team is always available and on standby to assist and swiftly help deal with any findings.
    
3. **Regular Code Reviews:**
    
    We conduct regular internal code reviews to ensure that any new code merged into the main branch adheres to our security standards. 
    
4. **Secure Development Practices:**
    
    We follow best practices in secure software development, including secure coding guidelines, dependency management, and vulnerability patching. We have implemented Continuous Integration to automatically build and test our codebase with every change, and utilize Unit Testing to verify the functionality and security of individual components of the wallet. We also use automated tools to perform regular security scans and static code analysis. 
    
5. **Trusted Cryptography Dependency:**
    
    The cryptography and keystore functionality of Gem Wallet is implemented using Trust Wallet Core, an open-source library that has been [independently audited](https://github.com/trustwallet/wallet-core/tree/master/audit). Trust Wallet Core is a well-established and widely used library in the crypto community, providing us with a robust and secure foundation for our wallet's cryptographic operations. [[Trust Wallet Core GitHub Repository](https://github.com/trustwallet/wallet-core)]
    
6. **Planned Security Audit:**
We are committed to a comprehensive security audit for our iOS and Android applications before the end of Q4 this year. This audit will be conducted by a reputable third-party security firm to ensure that our wallet meets the highest security standards.
7. **Experienced Development Team:**
    
    Our developers are highly experienced and dedicated professionals with extensive expertise in building secure and reliable wallets. Their deep understanding of blockchain technology and cryptography ensure that Gem Wallet is developed with the highest standards of security and in mind.
    
8. **User Education and Dedicated Support**
    
    While not directly linked to wallet security itself, we take user education just as seriously. We provide[ up-to-date documentation](https://docs.gemwallet.com/) and [a comprehensive learning section](https://gemwallet.com/learn/category/security/) to educate our users about best security practices for handling their wallets and crypto assets. Additionally, we have a dedicated support team ready to assist users with any questions or issues they might face.
    

Although we haven't undergone a formal security audit ourselves yet, we believe the measures outlined above demonstrate our commitment to security. Our open-source nature, combined with the use of audited dependencies like Trust Wallet Core and our proactive approach to security, provide a transparent and collaborative environment where security is continuously evaluated and improved.

### Has the wallet's smart contract code or security modules been audited?

No

### Does the wallet have an internal security team?

No

### Any other security testing that should be noted?

No

### Scam protection?

No, but we are looking at ways to integrate this feature.

### Does the wallet support connecting to Ethereum applications?

WalletConnect: https://docs.gemwallet.com/guides/walletconnect/

### Does the wallet support connecting to a hardware wallet?

No

### Does the wallet support importing Ethereum RPC endpoints?

No

### Does the wallet support viewing and interacting with NFTs?

No

### Does the wallet support staking directly?

Yes: https://docs.gemwallet.com/defi/stake/

### Does the wallet support swaps directly?

Yes: https://docs.gemwallet.com/defi/swap/

### Does the wallet support bridging directly?

No

### Does the wallet support multi-chain networks?

Yes: https://docs.gemwallet.com/chains/

### Does the wallet support Ethereum layer 2 networks?

Yes: https://docs.gemwallet.com/chains/

### Does the wallet allow the user to customize gas fees?

No

### Does the wallet support sending transactions to ENS addresses?

Yes, users can enter any ens address to send funds to another user. We support different name services for chains we support: https://docs.gemwallet.com/faq/naming/

### Does the wallet support importing or automatically querying and displaying ERC-20 tokens?

It supports both: https://docs.gemwallet.com/guides/add-token/

### Does the wallet support EIP-1559 (type 2) transactions?

Yes, we only support it.

### Does the wallet have any recovery options?

Only Manual: https://docs.gemwallet.com/guides/backup-wallet/

### Does the wallet have fiat on-ramps through credit/debit cards, wire transfers, or bank transfers (ACH)?

We support fiat on-ramps for credit/debit cards

### Does the wallet support withdrawals to fiat?

No, but its planned and coming soon

### Is the wallet a multi-signature wallet?

No

### Does the wallet support social recovery?

No

### Does the wallet support bundled transactions?

No

### Who can the ethereum.org team can contact regarding the wallet in future?

Telegram: https://t.me/magicwrex

### Does the wallet have a dedicated support team?

Our support staff answers fast (within a couple of hours)

You can join our Discord and open a ticket or email us directly (support@gemwallet.com)

### What educational resources/documentation do you provide to users?

Our docs are a living document that cover all aspects of Gem Wallet: https://docs.gemwallet.com/

We also have learn/blog section on our Website with more evergreen content: https://gemwallet.com/learn/

### Does the wallet have any integrated tools not mentioned above?

_No response_

### Would you like to work on this issue?

- [X] Yes
- [ ] No

### PR DESCRIPTION
**Pull request by wrexgem**
## Description

New PR to get Gem Wallet added to the Catalogue of Wallets on ethereum.org. See approved issue below. Thank you!

## Related Issue
- Fixes #13206

### CODE REVIEW

File: src/data/wallets/wallet-data.ts [hunk 1]
```diff
@@ -1859,6 +1860,66 @@ export const walletsData: WalletData[] = [
...
+    chromium: false,
+    hardware: false,
+    open_source: true,
+    repo_url: "https://github.com/gemwalletcom/",
+    non_custodial: true,
+    security_audit: [],
+    scam_protection: false,
+    hardware_support: false,
+    rpc_importing: false,
+    nft_support: false,
```
Comment (wackerow) [inline src/data/wallets/wallet-data.ts:1906]: https://gemwallet.com/nft-wallet/

False?

File: src/data/wallets/wallet-data.ts [hunk 2]
```diff
@@ -1859,6 +1860,66 @@ export const walletsData: WalletData[] = [
...
+    gas_fee_customization: false,
+    ens_support: true,
+    erc_20_support: true,
+    buy_crypto: true,
+    withdraw_crypto: false,
+    multisig: false,
+    social_recovery: false,
+    onboard_documentation: "https://docs.gemwallet.com/",
+    documentation: "https://docs.gemwallet.com/",
+    new_to_crypto: true,
```
Comment (wackerow) [inline src/data/wallets/wallet-data.ts:1921]: The wallets were recently updated wth a curated list for "new to crypto" to improve UX for beginners after much research. Let's not add this flag just yet; can be considered for the future (cc: @konopkja)

```suggestion
```

**Review by wackerow — CHANGES_REQUESTED** [review]
Thanks @wrexgem! Couple notes

File: src/data/wallets/wallet-data.ts (hunk 2 above)
Comment (konopkja) [reply src/data/wallets/wallet-data.ts:1921]: remove the tag

**Comment by wrexgem** [comment]
Added NFT tag and removed new to crypto tag. Should be good now!

**Review by wackerow — APPROVED** [review]
Looks good! Thanks @wrexgem

**Comment by wackerow** [comment]
@all-contributors please add @wrexgem for tool

**Pull request merged**